app/src/androidTest/java/com/github/sdp_begreen/begreen/models/CustomLatLngTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.github.sdp_begreen.begreen.models
+
+import com.google.android.gms.maps.model.LatLng
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+
+class CustomLatLngTest {
```

Comment:
This test class should not be in ´androidTest´ but rather in the ´test´ folder.

Reply:
You are absolutely right, thank you for noticing, this is a mistake on my part.